sites/postcss-preset-env/blog/postcss-bundler.md

```diff
@@ -0,0 +1,135 @@
+---
+title: PostCSS Bundler
+description: A standards compliant reference CSS bundler
+date: 2023-09-15
+---
+
+[PostCSS Bundler] is a standards compliant reference CSS bundler.
+
+It is built around `@import` statements, similarly to how [esbuild](https://esbuild.github.io), [postcss-import](https://www.npmjs.com/package/postcss-import) or [lightningcss](https://lightningcss.dev) bundle CSS.
+
+```css
+@import url('variables.css');
+@import url('typography.css');
+@import url('components/not-a-card.css');
+```
+
+
+## Why this is matters now
```

Comment:
```suggestion
## Why this matters now
```

Reply:
Unsure if this was intentional

Reply:
nope, probably was `why this is important now` in an earlier iteration :)